crates/feeder-gateway/src/main.rs

```diff
@@ -421,13 +425,29 @@ fn resolve_block(
         .map(|(_, r, _)| r.clone())
         .collect::<Vec<_>>();
 
-    let receipt_commitment = calculate_receipt_commitment(&receipts)?;
+    let receipt_commitment: ReceiptCommitment = calculate_receipt_commitment(&receipts)?;
```

Comment:
this change is probably unnecessary?

Reply:
Absolutely, I'll remove this change. Must be some auto-complete artifact...